README.md

```diff
@@ -317,3 +317,59 @@ You can run the project using Docker, sample docker-compose.yml file includes Bu
 3.  **Commit** changes to your own branch
 4.  **Push** your work back up to your fork
 5.  Submit a **Pull request** so that we can review your changes
+
+## Sitecore.Foundation.Extentions Project (Now included in main Solution)
+1. This includes build-in extention method functionality for client's wanting to integrate with a Tranditional Sitecore XM/XP instance implementation 
+    - These extension methods can be access via the following files under the src\Middleware\src\SitecoreExtensions\code folder:
```

Comment:
Thanks for adding this documentation. Would you mind running this through a spellcheck and grammar tool? I'm seeing quite a few errors. For example:

- build-in should be built-in
- extention should be extension
- client's should be clients
- Tranditional should be Traditional

Reply:
Resolved the BAC-3584 issue and fixing Seo meta tagging for products

Reply:
Resolved this request.